data/settings/1.14.x/pki.toml

```
@@ -0,0 +1,37 @@
+[[docs.ref.pki_bundle-name_data]]
+name_override = "<bundle-name>.data"
+description = "A Base64-encoded PEM-formatted certificate bundle; This setting can contain more than one certificate."
```

Not at English expert, but should this be?

```suggestion
description = "A Base64-encoded PEM-formatted certificate bundle; this setting can contain more than one certificate."
```